Classes/View Controllers/RootViewControllers.swift

```diff
@@ -49,3 +49,13 @@ func newSearchRootViewController(client: GithubClient) -> UIViewController {
     nav.tabBarItem.selectedImage = UIImage(named: "tab-search-selected")
     return nav
 }
+
+func newBookmarksRootViewController(client: GithubClient) -> UIViewController {
+    let title = NSLocalizedString("Bookmarks", comment: "")
+    let controller = BookmarksViewController(client: client)
+    controller.title = title
+    let nav = UINavigationController(rootViewController: controller)
+    nav.tabBarItem = UITabBarItem(tabBarSystemItem: .bookmarks, tag: 3)
```

Comment:
Hmm, I'd love to use GitHub's official iconography. There's a `bookmark` image already. Let's use that! Then we don't have to init a new item either.